test/setup.js

```diff
@@ -20,6 +20,8 @@ const days = eachDay(from, subDays(to, 1)) // last day should not be included (h
 const historicDataTo = subDays(endOfYesterday(), 200)
 const historicDataFrom = startOfDay(subDays(historicDataTo, 205))
 
+const address = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
```

Comment:
can you rename to something more descriptive like `eth_address`?

Reply:
Alright, I'll do it